tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/EmailSendMethod.scala

```diff
@@ -201,6 +213,10 @@ class EmailSendMethod @Inject()(serializer: EmailSendSerializer,
     }
   }
 
+  def toMimeMessage(name: String, message: Message): Try[MimeMessageWrapper] = {
+    ??? //todo
```

Comment:
How can I convert from `org.apache.james.mime4j.dom.Message` to `org.apache.james.server.core.MimeMessageWrapper` (or `javax.mail.internet.MimeMessage`) ?

Reply:
Use DefaultMessageWriter to be writing the bytes of your messages, then write a `MimeMessageSource` backed by a byte array :-)